Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/CameraIntegrationTest.java

```diff
@@ -83,7 +83,8 @@
 	@ClassRule
 	public static GrantPermissionRule grantPermissionRule = GrantPermissionRule.grant(
 			Manifest.permission.WRITE_EXTERNAL_STORAGE,
-			Manifest.permission.READ_EXTERNAL_STORAGE);
+			Manifest.permission.READ_EXTERNAL_STORAGE,
```

Comment:
I cannot comment on the line directly because it is not part of the set of changes, but is there a reason why you catch the exception that can occur while creating the picture file (see code snippet below). because what I have found is that just having the `setUp()` method throw the exception gives you the stacktrace plus the underlying reason why this failed opposed to just the message you put into the call to `fail(..)`. 
```
catch(IOException e) {
    fail("Picture file could not be created.");
}
```

Reply:
You are right, that is bs, there is no reason for this try-catch to exist.